src/docdb/tree/DocDBDocumentTreeItem.ts

```diff
@@ -18,24 +19,22 @@ export class DocDBDocumentTreeItem implements IAzureTreeItem {
     public readonly commandId: string = 'cosmosDB.openDocument';
 
     public readonly partitionKeyValue: string | undefined;
+    public label: string;
```

Comment:
private _label:string;

public get label() { 
  return this._label;
etc.

ditto for next class

Reply:
The update operation for the collection view in Mongo would have the side-effect of updating a document's label too. In order for that to work, the label field needs a setter. After the offline discussion, it looks like a private+getter+setter is preferred as opposed to public.